.babelrc

```diff
@@ -0,0 +1 @@
+{  "presets": ["@babel/preset-env"]}
```

Comment:
For what are you using babel ?

Reply:
Some ES6 syntax somewhere I believe :thinking: I'm not sure anymore :laughing:
I remember it had to do with some compatibility issues of ES6

Reply:
Remove this babel file, its seems unnecessary like we talk by slack